resources/web/docs_js/index.js

```diff
@@ -281,3 +282,8 @@ $(function() {
 
   // Test comment used to detect unminifed JS in tests
 });
+
+// Tabbed widgets
+switchTabs();
+setActiveTab();
+changeTabs();
```

Comment:
This is a stab in the dark, but it works. I tried making one massive function, but that didn't work. I also tried invoking the functions in `./components/tabbed_widget`, but that didn't work.